plugin-hrm-form/src/utils/mappers.ts

```diff
@@ -49,11 +49,14 @@ export const mapChannelForInsights = (channel: string) => {
   }
 };
 
-export const mapAge = (age: string) => {
+export const mapAge = (ageOptions: string[]) => (age: string) => {
   const ageInt = parseInt(age, 10);
 
-  if (ageInt >= 0 && ageInt <= 25) return age;
-  if (ageInt > 25) return '>25';
+  const maxAge = ageOptions.find(e => e.includes('>'));
+  const maxAgeInt = parseInt(maxAge.replace('>', ''), 10);
+
+  if (ageInt >= 0 && ageInt <= maxAgeInt) return age;
+  if (ageInt > maxAgeInt) return maxAge;
```

Comment:
Maybe we want to make this less fragile. Right now, if `>xx` isn't specified in configuration, `maxAge.replace(...)` will throw an error and fail. Could we do something like:
```suggestion
  const maxAge = ageOptions.find(e => e.includes('>'));
  if (maxAge) {
    const maxAgeInt = parseInt(maxAge.replace('>', ''), 10);
    if (ageInt > maxAgeInt) return maxAge;
  }
  if (ageInt >= 0 && ageInt <= maxAgeInt) return age;
```
Does that have problems that I'm missing?